### PR TITLE
Feat: Edit Note and added toast

### PR DESCRIPTION
**Pull request by sainath7878**
Files to check: 

- [src/components/displayNote/DisplayNote.jsx](https://github.com/sainath7878/joddkeep/compare/editNote&Label?expand=1#diff-357a6e67d3574cc3d5573aead657eac1e344d4d7a44df0bf5443fd53bdc79ba7)
- [src/components/editNote/EditNote.jsx](https://github.com/sainath7878/joddkeep/compare/editNote&Label?expand=1#diff-ed9d5f35d044240177cb338a55afcae5d06f07f2e61a09e83a875b38e1a1b117)
- [src/components/editNote/editNote.css](https://github.com/sainath7878/joddkeep/compare/editNote&Label?expand=1#diff-231296e570cce8aece5e19053687f2cae5638bfb89c29586d17eb7effde029dc)
- [src/components/toast/Toast.jsx](https://github.com/sainath7878/joddkeep/compare/editNote&Label?expand=1#diff-81829b4c685b73fd657a1c35eb8eaebd62da9c7ab4109ae52c3ef93972b0b85d)
- [src/context/notesContext.js](https://github.com/sainath7878/joddkeep/compare/editNote&Label?expand=1#diff-f0b61a0825b8ed568eed5811cb2ebf19e5e50845327123432fcde8d229d72661)

### CODE REVIEW

**Review by Vishnu-Aithal — APPROVED** [review]
All the functionalities are working . Code is Good and Readable
small Suggestions-
toast could be maintained in different context
exit edit-mode without saving changes

File: src/components/toast/toast.css [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+  z-index: var(--Z-INDEX-20);
+}
+
+.show {
+  transform: translateX(0);
+}
+
+.hide {
+  transform: translateX(150%);
+}
```
Comment (BalajiHarinath) [inline src/components/toast/toast.css:15]: Appreciation :- Nice use of translate to show the toast and hide it.

File: src/components/toast/Toast.jsx [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+        type: "SET_TOAST",
+        payload: {
+          state: false,
+          msg: "",
+          type: "",
+        },
+      });
+    }, 3000);
+    return () => clearTimeout(timeOut);
+  }, [authDispatch]);
```
Comment (BalajiHarinath) [inline src/components/toast/Toast.jsx:25]: Suggestion:- Can remove the dependency array and just add an empty dependency array.

**Review by BalajiHarinath — APPROVED** [review]
Nice UI and your code is clean.

File: src/components/toast/Toast.jsx (hunk 2 above)
Comment (sainath7878) [reply src/components/toast/Toast.jsx:25]: vsCode gives a warning to include authDispatch

**Pull request merged**